src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+@import '../styles/variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: 166px;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+  }
+
+  &__title {
+    font-size: $font-size-base;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 $small-margin;
```

Comment:
The vertical margin '40px' seems to be a unique value. If this is a standard spacing used in your project, consider creating a variable for it to maintain consistency.